coreneuron/logging/easylogging.h

```diff
@@ -0,0 +1,6663 @@
+//
```

Comment:
We don't favor simply including external libraries. Much better would be to add git submodules for such libraries